package.json

```diff
@@ -23,7 +23,7 @@
     "test:journey:k3d": "k3d cluster delete pepr-dev && k3d cluster create pepr-dev --k3s-arg '--debug@server:0'",
     "test:journey:build": "npm run build && npm pack",
     "test:journey:image": "docker buildx build --tag pepr:dev . && k3d image import pepr:dev -c pepr-dev",
-    "test:journey:run": "jest journey/entrypoint.test.ts",
+    "test:journey:run": "jest --detectOpenHandles journey/entrypoint.test.ts",
     "test:journey:run-wasm": "jest journey/entrypoint-wasm.test.ts",
```

Comment:
The WASM journey test will need this too because it basically follows the same logic as `test:journey:run` but does `npx pepr build -r` to add a custom repo.